### PR TITLE
Fixes #3156-Added WhatsApp,Telegram links and Fixed Alignment in blog section

### LINKED ISSUE #3156: Social Icons in Blog section requires attention
**Actual Behaviour**
There are a few icons in social section also there is no space in between them.
Also the cursor doesn't show any responsive change as we hover over them

**Expected Behaviour**

Share to whatsapp, Telegram should be available

**Would you like to work on the issue?**

YES
**ScreenShot:**
Screenshot 2019-12-27 at 9 59 46 PM

### PR DESCRIPTION
**Pull request by anishagg17**
Fixes #3156

Changes: 
<ul>
<li>Added WhatsApp,Telegram links</li>
<li>Fixed Alignment</li>
<li>Made them responsive with respect to device</li>
<li>Cursor turns to pointer on hover</li>
</ul>
 <a href="https://pr-3157-fossasia-susi-web-chat.surge.sh">
Demo Link</a> "https://pr-3157-fossasia-susi-web-chat.surge.sh"

Screenshots of the change:

Screenshot 2020-01-02 at 12 29 48 AM
Screenshot 2020-01-02 at 12 29 45 AM

### CODE REVIEW

**Comment by anishagg17** [comment]
> Give some padding between the buttons in the mobile view. Rest looks fine 👍

Done Sir

**Comment by anishagg17** [comment]
> Squash your commits.

Done Sir

**Comment by anishagg17** [comment]
@akshatnitd please review

File: src/components/About/Blog/index.js [hunk 1]
```diff
@@ -137,9 +141,15 @@ const BlogNavigation = styled.div`
 `;
 
 const SocialButtons = styled.div`
-  width: 100%;
+  width: 30%;
   display: flex;
   padding: 0.625rem 0 1.25rem 0.625rem;
+  align-items: center;
+  justify-content: space-between;
```
Comment (akshatnitd) [inline src/components/About/Blog/index.js]: Do not use `space-between` here. Use `flex-start` and let the width be `100%`. The spacing is excessive in your current implementation.

**Comment by akshatnitd** [comment]
@anishagg17 Please write proper commit message.

**Comment by anishagg17** [comment]
> @anishagg17 Please write proper commit message.

Done Sir!

**Pull request merged**